app/models/vim_performance_state.rb

```diff
@@ -203,6 +205,11 @@ def self.capture_tag_names(obj)
     obj.tag_list(:ns => "/managed").split.join("|")
   end
 
+  def self.capture_image_tag_names(obj)
+    return nil unless obj.respond_to?(:container_image)
+    obj.container_image.tag_list(:ns => "/managed").split.join("|")
```

Comment:
Acquiring container images is going through changes so im going to go on the safe side here and change this to:
```Ruby
 return '' unless obj.respond_to?(:container_image) && obj.container_image.present?
```